obspy/core/utcdatetime.py

```diff
@@ -981,63 +987,63 @@ def __unicode__(self):
         """
         return str(self.__str__())
 
+    def _operate(self, other, op_func):
+        if isinstance(other, UTCDateTime):
+            a = py3_round(self._ns, self.precision - 9)
+            b = py3_round(other._ns, self.precision - 9)
```

Comment:
Should this both `self.precision` arguments be replaced with `min(self.precision, other.precision)`? Or maybe raise a warning if two UTCDateTime objects of differing precision are used as that operation is not really defined?

Reply:
Good thought, I like the idea of raising a warning, maybe an error eventually. I will use the min of the precisions and raise a warning if they are different.